com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -688,42 +756,65 @@ public UpscalingFilterSelection upscalingFilter
             set { m_UpscalingFilter = value; }
         }
 
+        /// <summary>
+        /// Specifies the <c>LightRenderingMode</c> for the main light used by this <c>UniversalRenderPipelineAsset</c>.
+        /// </summary>
+        /// <see cref="LightRenderingMode"/>
         public LightRenderingMode mainLightRenderingMode
         {
             get { return m_MainLightRenderingMode; }
             internal set { m_MainLightRenderingMode = value; }
         }
 
+        /// <summary>
+        /// Specifies if shadows for the main light should be used for this <c>UniversalRenderPipelineAsset</c>.
```

Comment:
You can add the "for this <c>UniversalRenderPipelineAsset</c>" if you want, but I generally find that redundant and repetitive information. No strong opinion though.
```suggestion
        /// Specifies if objects lit by main light cast shadows.
```